traverson4j-hc5/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
Whats this do?

Reply:
enables mocking of final classes in Mockito https://www.baeldung.com/mockito-final